test/test_torch.py

```diff
@@ -2101,6 +2101,15 @@ def test_inference(default_dtype):
         test_inference(torch.float64)
         test_inference(torch.float32)
 
+    def test_qengnie(self):
+        qengines = torch._C._supported_qengines()
+        # [TODO] Enable after the interface change
+        # original_qe = torch._C._get_qengine()
+        # for qe in qengines:
```

Comment:
`qe` is an int but set_qengine expects `at::QEngine`. Will this work?

Reply:
it doens't work right now that's why I commented them out. it should work after the interface change. we should be returning/pass `int` in `_get_qengine` and `_set_qengine` as well